javalib/src/main/scala/java/util/concurrent/atomic/AtomicBoolean.scala

```diff
@@ -1,16 +1,34 @@
 package java.util.concurrent.atomic
 
+import scala.scalanative.native
+import scala.scalanative.native.Atomic
+
 class AtomicBoolean(private[this] var value: Boolean) extends Serializable {
   def this() = this(false)
 
+  private val lock: native.Ptr[native.CInt] = native.stackalloc[native.CInt]
```

Comment:
The correct solution to this would be to have a private field `lock` and then synchronize on it (by using address of the field). Unfortunately, we don't have APIs to support this use case at the moment.